src/validator.js

```diff
@@ -14,7 +14,8 @@ const checkJsonPath = require('./lib/json-path-errors');
 const missingTransitionTarget = require('./lib/missing-transition-target');
 
 function formatError(e) {
-  return e.Code ? `${e.Code}: ${e.Message}` : e.message;
+  const code = e.Code ?? e['Error code'];
```

Comment:
The nullish coalescing operator (`??`) has been added in Node 14. As this library is supposed to run with Node 10 (see [here](https://github.com/ChristopheBougere/asl-validator/blob/master/.github/workflows/nodejs.yml#L16)), could you replace it with a legacy syntax?